react/Modal/index.jsx

```diff
@@ -63,13 +62,11 @@ class ModalContent extends Component {
     this.scrollingContent.addEventListener('scroll', this.handleScroll, {
```

Comment:
@GoOz Does this line still make sense without the scroll lock? I think so, but not quite sure.

Reply:
Yes, it has another purpose.